packages/block-library/src/index.js

```diff
@@ -59,8 +59,9 @@ import * as textColumns from './text-columns';
 import * as verse from './verse';
 import * as video from './video';
 import * as tagCloud from './tag-cloud';
-
 import * as classic from './classic';
+import * as socialLinks from './social-links';
+import * as socialLink from './social-links/social-link';
```

Comment:
Can we have Social Link block in the top-level folder as well to make their discovery easier?

Reply:
My idea was to keep the social-link block as a child of social-links, so you wouldn't insert a single social-link but use them only as the inner blocks for social-links block.

There are a couple of different ways to solve, we could also just have a single social-link block, with no parent block, and then they are inserted multiple times using columns to control layout.

I felt that having the block with multiple fits more in line with how people use social links as a distinct block with multiple items.

I'm open to any suggestions here.

Reply:
It's what we have at the moment for
- `columns` + `columns`
- `navigation-menu` + `navigation-menu-itme`

See https://github.com/WordPress/gutenberg/tree/master/packages/block-library/src.

It's mostly for consistency and to have an easy way to discover all blocks with tooling. This way you can easily identify all `block.json` files with a simple pattern match.

Reply:
:+1: Moved in 5668a9a56